tests/unit/store.t

```diff
@@ -8,7 +8,7 @@ use Log::Any::Adapter 'TAP';
 
 use ProductOpener::Store qw/:all/;
 
-is( get_fileid('Do not challenge me!'), 'do-not-challenge-me' );
+is( get_fileid('Do not challenge me!', 0, 'en'), 'do-not-challenge-me' );
```

Comment:
We have to keep this test as is, because there are many places where we call get_fileid with only first arguments, and this has to work. (The real fix is that you put `undef` as defaults in function signature).

```suggestion
is( get_fileid('Do not challenge me!'), 'do-not-challenge-me' );
```